File: Modul 5/soal_4/PRAK504-2410817310003-Muhammad_Guntur_Ricky_Adhitya.py
def reverse(Bil):
    reverse_num = 0
    temp = 0
    v = 10
    while v < Bil*10 : 
        reverse_num *= 10
        reverse_num += ((Bil%v)-temp)/(v/10)
        temp = Bil%v
        v *= 10
    return int(reverse_num)

A, B = map(int, input().split())
A = reverse(A)
B = reverse(B)

C = A + B
print(reverse(C))

File: Modul 5/soal_4/PRAK504-2410817310003-Muhammad_Guntur_Ricky_Adhitya.c
#include <stdio.h>

int reverse(int Bil){
    int reverse_num = 0;
    int temp = 0;
    for(int v = 10; v < Bil*10; v *= 10){
        reverse_num *= 10;
        reverse_num += ((Bil%v)-temp)/(v/10);
        temp = Bil%v;
    }
    return reverse_num;
}

int main() {
    int A, B;
    scanf("%d %d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C = A+B;
    printf("%d",reverse(C));
}